### ISSUE TITLE
Download doesn't work on iOS since Nativescript 6.0

### ISSUE THREAD

**Issue by Beckmi2** [NativeScript/NativeScript#7654]
Problem is already described here including sample project:
https://github.com/triniwiz/nativescript-downloader/issues/38

**Environment**
Provide version numbers for the following components (information can be retrieved by running `tns info` in your project folder or by inspecting the `package.json` of the project):
 - CLI: 6.0.2
 - Cross-platform modules:  6.0.4
 - Android Runtime: 6.0.0
 - iOS Runtime: 6.0.1
 - Plugin(s):
"dependencies": {
"@proplugins/nativescript-dom": "^2.1.0",
"@proplugins/nativescript-globalevents": "^1.3.0",
"@proplugins/nativescript-orientation": "^2.3.1",
"@proplugins/nativescript-purchase": "^2.1.1",
"fs-copy-file-sync": "1.1.1",
"moment": "2.24.0",
"nativescript-downloader": "2.1.4",
"nativescript-local-notifications": "3.2.1",
"nativescript-plugin-firebase": "9.0.2",
"nativescript-sqlite": "latest",
"nativescript-theme-core": "1.0.6",
"nativescript-ui-listview": "7.0.0",
"nativescript-ui-sidedrawer": "7.0.0",
"rxjs": "6.5.2",
"tns-core-modules": "~6.0.0"
},
"devDependencies": {
"eslint": "5.16.0",
"nativescript-dev-webpack": "~1.0.0",
"node-sass": "4.12.0"
},

**Describe the bug**
Download file doesn't work on IOS since Nativescript 6.0. On Android all is fine.

**To Reproduce**
Menu 'Datenabgleich', button 'Download' should start the download.

**Sample project**
[Downloader.zip](https://github.com/NativeScript/NativeScript/files/3477193/Downloader.zip)

**Additional context**
see also [https://github.com/triniwiz/nativescript-downloader/issues/38](url)

**Comment by NickIliev** [comment]
Closing as a duplicate to https://github.com/triniwiz/nativescript-downloader/issues/38

@Beckmi2 I think the issue in the nativescript-downloader repository is enough for now. I would recommend posting the error log (that you are receiving) in the linked issue.

As the `package.json` is containing paid plugin it is not possible for us to test the issue on our side so again consult with the author of `nativescript-downloader`. If you believe the issue is related to NativeScript please provide a project that doesn't contain paid plugins.

**Comment by Beckmi2** [comment]
I didn’t know, that there is a disadvantage for me signing up to proplugins :-(
The problem is that I can’t develop my app further because I upgraded to Nativescript 6.0 and no things that worked before don’t work now.  That was the reason I turned also directly to the nativescript team.
The only reason I included all plugins used in my original app was because plugins often influence each other.
I will try to get an answer from the plugin’s author, but still now there was no echo ...
